<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopSwitchConfiguration</class>
 <widget class="QDialog" name="DesktopSwitchConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesktopSwitch settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QSpinBox" name="rowsSB">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelButtons">
        <property name="text">
         <string>Desktop labels:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelRowsSB">
        <property name="text">
         <string>Number of rows:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="labelTypeCB">
        <item>
         <property name="text">
          <string>Numbers</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Names</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QCheckBox" name="showOnlyActiveCB">
        <property name="toolTip">
         <string>If checked, widget will be just an indicator</string>
        </property>
        <property name="text">
         <string>Show only active desktop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="namesGroupBox">
     <property name="title">
      <string>Desktop names</string>
     </property>
     <layout class="QFormLayout" name="formLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
